### PR TITLE
[Docs] Move over docs from poktroll-docker-compose-example

### PR DESCRIPTION
**Pull request by Olshansk**
## Summary

Make it easier to onboard users to Shannon by providing a nicer skin on the docs in [poktroll-docker-compose-example](https://github.com/pokt-network/poktroll-docker-compose-example)

Also:
- Improve ToC
- Improve introduction
- Add a results video

Screenshot 2024-08-23 at 11 23 38 AM

## Type of change

Select one or more:

- [ ] New feature, functionality or library
- [ ] Bug fix
- [ ] Code health or cleanup
- [ ] Documentation
- [ ] Other (specify)

## Testing

- [x] **Documentation**: `make docusaurus_start`; only needed if you make doc changes
- [ ] **Unit Tests**: `make go_develop_and_test`
- [ ] **LocalNet E2E Tests**: `make test_e2e`
- [ ] **DevNet E2E Tests**: Add the `devnet-test-e2e` label to the PR.

### CODE REVIEW

**Review by okdas — APPROVED** [review]
This looks good. I looked through the docs quickly.
1. Do you want me to try this on a fresh server on vultr?
2. Do you think we should add the ufw rule we discovered last week (the one blocking connections from the appgate to relayminer on the same machine)?

**Comment by Olshansk** [comment]
@okdas 

> Do you want me to try this on a fresh server on vultr?

Yes, would appreciate it.

> Do you think we should add the ufw rule we discovered last week (the one blocking connections from the appgate to relayminer on the same machine)?

There's a lot more than that. Added things back.

File: docusaurus/docs/operate/quickstart/docker_compose_debian_cheatsheet.md [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+
+And then install docker:
+
+```bash
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin
+```
+
+## Retrieve the source code
+
+Then install docker-compose
```
Comment (okdas) [inline docusaurus/docs/operate/quickstart/docker_compose_debian_cheatsheet.md:86]:
```suggestion
Then pull the github repo
```

**Comment by Olshansk** [comment]
Merging with `sed` error because it is expected given the circumstance.

Screenshot 2024-08-25 at 1 02 16 PM

**Pull request merged**